docs/node_scenarios.md

```diff
@@ -25,7 +26,7 @@ Following node chaos scenarios are supported:
 A google service account is required to give proper authentication to GCP for node actions. See [here](https://cloud.google.com/docs/authentication/getting-started) for how to create a service account.
  
 **NOTE**: A user with 'resourcemanager.projects.setIamPolicy' permission is required to grant project-level permissions to the service account.
- 
+ r
```

Comment:
need to remove this extra line

Reply:
I've fixed it along with a few other minor changes.